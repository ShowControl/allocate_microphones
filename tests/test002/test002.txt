[show test002]  // one actor, one character
Designer=John Sauter
DateNumeric=2019-09-22
DateText=September 22, 2019

[character character_01]
[character character_02]

[actor actor_01]
[actor actor_02]

[role]
actor=actor_01
character=character_01
[role]
actor=actor_02
character=character_02

[page 1]
[act 1]
[scene 1]
[enter character_01]
[page 2]
[exit character_01]
[enter character_02]
[page 3]
[exit character_02]

[end show]

