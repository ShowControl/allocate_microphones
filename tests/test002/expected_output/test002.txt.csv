page, "actor_01", "actor_02"
"1","*"," "
"2"," ","*"
"3"," "," "
